reportportal_client/service_async.py

```diff
@@ -130,12 +131,16 @@ def __init__(self, endpoint, project, token, api_base="api/v1",
                 during items processing (in thread)
             is_skipped_an_issue: option to mark skipped tests as not
                 'To Investigate' items on Server side.
+            verify_ssl: option to not verify ssl certificates
```

Comment:
This comment isn't clear.
Will we not verify the certificate when turning on this flag (set it to true)?